mne/simulation/metrics.py

```diff
@@ -0,0 +1,68 @@
+# Authors: Yousra Bekhti <yousra.bekhti@gmail.com>
+#          Mark Wronkiewicz <wronk@uw.edu>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from scipy.linalg import norm
+
+# TODO: Add more localization accuracy functions. For example, distance between
+#       true dipole position (in simulated stc) and the centroid of the
+#       estimated activity.
+
+
+def _check_stc(stc1, stc2):
+    """Helper for checking that stcs are compatible"""
+    if stc1.data.shape != stc2.data.shape:
+        raise ValueError('Data in stcs must have the same size')
+    if np.all(stc1.times != stc2.times):
+        raise ValueError('Times of two stcs must match.')
+
+
+def source_estimate_quantification(stc1, stc2, metric='rms'):
+    """Helper function to calculate matrix similarities.
+
+    Parameters
+    ----------
+    stc1 : SourceEstimate
+        First source estimate for comparison.
+    stc2 : SourceEstimate
+        First source estimate for comparison.
+    metric : str
+        Metric to calculate, 'rms' or 'cosine'.
+
+    Returns
+    -------
+    score : float | array
+        Calculated metric.
+
+    Notes
+    -----
+    Metric calculation has multiple options:
+
+        * rms: Root mean square of difference between stc data matrices.
+        * cosine: Normalized correlation of all elements in stc data matrices.
+
+    Notes
+    -----
+    .. versionadded:: 0.10.0
+    """
+    known_metrics = ['rms', 'cosine']
+    if metric not in known_metrics:
+        raise ValueError('metric must be a str from the known metrics: '
+                         '"rms" or "cosine"')
+
+    # This is checking that the datas are having the same size meaning
+    # no comparison between distributed and sparse can be done so far.
+    _check_stc(stc1, stc2)
+    data1, data2 = stc1.data, stc2.data
+
+    # Calculate root mean square difference between two matrices
+    if metric == 'rms':
+        return np.sqrt(np.mean((data1 - data2) ** 2))
+
+    # Calculate correlation coefficient between matrix elements
+    elif metric == 'cosine':
+        score = (np.correlate(data1.flatten(), data2.flatten()) /
+                 (norm(data1) * norm(data2)))
+        return 1 - score
```

Comment:
1 -> 1.

I would just be explicit

score = np.dot(data1.flatten(), data2.flatten()) / (norm(data1) * norm(data2))